build-system/babel-plugins/babel-plugin-transform-jss/index.js

```diff
@@ -36,17 +36,39 @@
  * ```
  */
 
+const crypto = require('crypto');
 const {create} = require('jss');
 const {default: preset} = require('jss-preset-default');
+const {relative, join} = require('path');
 
 module.exports = function ({types: t, template}) {
   function isJssFile(filename) {
     return filename.endsWith('.jss.js');
   }
 
-  function compileJss(JSS) {
-    const jss = create();
-    jss.setup(preset());
+  const seen = new Set();
+  function compileJss(JSS, filename) {
+    const relativeFilepath = relative(join(__dirname, '../../..'), filename);
```

Comment:
This is a good question too: would the paths be different on Windows vs Unix due to dir separator? Or any other file encoding nuances?

Reply:
Many of our babel tests are currently not cross-OS compatible. I plan on addressing this in a future PR. I plan on fixing them in a future PR now that I dev more on windows